client/src/containers/Dashboard/Dashboard.js

```diff
@@ -37,6 +38,9 @@ class Dashboard extends React.Component {
         this.setState({loading: false});
       }
     });
+    fetch('jenkins').then((data) => {
+      this.jenkinsUrl = data.body.url;
```

Comment:
It's better to store this information in Redux so all components can access to this information without passing it as props to every component/container. 

To add this to redux, we only need to add an `action` to storing this information and adding this new property in the redux state in `reducer.js`.